### PR TITLE
12 created components to handle invalid paths

### PR DESCRIPTION
**Pull request by alxjnr**
the three new components display information relative to the invalid path. My commenting functionality has some conditional logic to handle posting empty comments/commenting without being logged in.

### CODE REVIEW

File: src/components/InvalidReview.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { useNavigate } from "react-router-dom";
+
+const InvalidReview = () => {
```
Comment (lewispricey) [inline src/components/InvalidReview.jsx:3]: Great use of a seperate component for an invalidReviewID, good choice of error message too 😁  With a useful button to go back to the main reviews

File: src/App.js [hunk 2]
```diff
@@ -27,6 +28,7 @@ function App() {
         />
         <Route path="/categories" element={<ListOfCategories />} />
         <Route path="/categories/:category_name" element={<ViewCategory />} />
+        <Route path="/*" element={<PageNotFound />} />
```
Comment (lewispricey) [inline src/App.js:31]: Great way to catch any rogue paths 👍

**Review by lewispricey — APPROVED** [review]
Awesome work on catching these errors Alex, Really like that you've used seperate components to keep this errror logic together.

I did have some trouble signing in, not sure if I'm missing something but tried a few usernames and couldnt get this to work. (It may just be me though 😆) For the portfolio piece it may be worth having a dropdown or somewhere to select a user just for any recruiters trying it out.

Good to merge when you're ready 🚀

File: src/App.js (hunk 2 above)
Comment (alxjnr) [reply src/App.js:31]: > my api hasn't been accessible for the last half an hour or so for some reason, maybe it was that, but will definitely put in a drop down box/default user! thank you

**Pull request merged**